brave/brave-core
# Android leo autocomplete

Resolves https://github.com/brave/brave-browser/issues/34665



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Type something in the URL bar.
2. Make sure there is an autocomplete suggestion to use Leo if it's enabled in settings.
3. Tap on the suggestion and make sure Leo chat opens with the suggestion response.

![Screenshot_20240111_132326](https://github.com/brave/brave-core/assets/12011303/9ee8a92a-1ada-4b81-938d-94321ed2944d)
![Screenshot_20240111_132633](https://github.com/brave/brave-core/assets/12011303/1305ddd5-c91e-4b00-8635-fba499bd1d95)
![Screenshot_20240111_132838](https://github.com/brave/brave-core/assets/12011303/8730b117-a4d1-497d-90d7-eeb27e46cb5f)
